### PR TITLE
fix gqlize - 2nd degree of update mutation does not work (add/update/delete/remove) 

### PR DESCRIPTION
**Pull request by jdulayvostronet**
fikx gqlize - 2nd degree of update mutation does not work

### CODE REVIEW

File: __tests__/mutation.test.js [hunk 1]
```diff
@@ -1073,3 +1073,274 @@ test("add multiple ids", async() => {
   const res2 = await graphql(schema, mutation2, undefined, undefined, variableValues);
```
Comment (jdulayvostronet) [inline __tests__/mutation.test.js:1073]: @all Kindly check if i have missing test cases so i can adjust accordingly :)

**Review by aerborne — APPROVED** [review]

**Review by christianbalderrama — APPROVED** [review]
LGTm

**Review by Azerothian — APPROVED** [review]

**Comment by Azerothian** [comment]
you need to modify the pull request to merge into the VostroNet/gqlize/master repo

**Comment by Azerothian** [comment]
https://github.com/VostroNet/gqlize/pull/1

**Comment by jdulayvostronet** [comment]
Wow thank you sir

**Comment by Azerothian** [comment]
you might want to close this one :D

**Pull request closed without merging**